src/go/rpk/pkg/cli/cmd/wasm/common.go

```diff
@@ -60,7 +60,7 @@ func ExistingTopic(admin sarama.ClusterAdmin, topic string) (bool, error) {
 }
 
 func CreateDeployMsg(
-	name string, description string, content []byte,
+	name string, filename string, description string, content []byte,
```

Comment:
If we're moving away from using the actual filename as the ID, the variable shouldn't be named `filename`. I could see myself coming back to this code in a few months and thinking "Ok, this is the filename" haha. How about `id`, and `--id`? It's clearer in my opinion.

Reply:
I think i'm just going to get rid of filename, it isn't necessary anyway